### PR TITLE
Increase timeout length for muffet

### PR DESCRIPTION
**Pull request by theosanderson**
Resolves(?) #152

### CODE REVIEW

**Comment by theosanderson** [comment]
I don't really see what is wrong with my original version here? I think trying to test for Loculus stuff that takes too long, if that's your goal, doesn't make sense on preview servers which can be throttled

**Comment by theosanderson** [comment]
(but I also don't have a problem with trying it, now you've got it passing)

**Comment by theosanderson** [comment]
The downside is just the code duplication

File: .github/workflows/broken-link-check.yml [hunk 1]
```diff
@@ -44,8 +44,8 @@ jobs:
         if: github.event_name == 'push' || github.event_name == 'pull_request'
         run: |
           echo "Waiting for 180 seconds to allow preview to be up..."
-          sleep 180
-  broken-link-checker:
+          # sleep 180
```
Comment (corneliusroemer) [inline .github/workflows/broken-link-check.yml:47]:
```suggestion
          sleep 180
```

**Pull request merged**